### PR TITLE
Export MutableFile

### PR DESCRIPTION
**Pull request by super-v-2038**
Thank you for your work! This package has saved me a lot of time.

It seems that `MutableFile` has not been exported, which may be an oversight.
If this is due to design, I apologize for taking up your time.

### CODE REVIEW

File: lib/mega.js [hunk 1]
```diff
@@ -1,7 +1,9 @@
 import { megaEncrypt, megaDecrypt, megaVerify } from './crypto/index.mjs'
 import Storage from './storage.mjs'
+import MutableFile from './mutable-file.mjs'
 import File from './file.mjs'
 import API from './api.mjs'
+import MutableFile from './mutable-file.mjs'
```
Comment (qgustavor) [inline lib/mega.js:6]: Imported it twice?

**Comment by qgustavor** [comment]
It was not an oversight: I considered exporting it but I didn't see any use case which would be useful, as you create MutableFiles usually using Storage. But I don't mind changing it.

**Review by qgustavor — CHANGES_REQUESTED** [review]
Can you check and fix the double import?

**Comment by super-v-2038** [comment]
According to the document, `MutableFile` seems to be a folder. Did I misunderstand it?

File: lib/mega.js (hunk 1 above)
Comment (super-v-2038) [reply lib/mega.js:6]: Oh my goodness, how could I write it twice

Comment (qgustavor) [reply lib/mega.js:6]: It happens sometimes, people are people and people make mistakes, that's why there are code reviews 😄

**Comment by qgustavor** [comment]
Actually both `File` and `MutableFile` files, if I were to name them, should be properly named `Node` and `MutableNode`, as they can be either files or folders. But since the original project named them files, it was kept because backward compatibility reasons.

**Pull request merged**